src/App.js

```diff
@@ -1,26 +1,9 @@
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 import './App.css';
 
 function App() {
   return (
```

Comment:
- [ ] The `App.js` component should be class-based. Kindly make use of a class component as it is part of the project requirement.

Reply:
I didn't know I had to change App component also. Thanks for pointing it out.